db_stress_tool/db_stress_test_base.cc

```diff
@@ -3411,7 +3412,22 @@ void StressTest::Open(SharedState* shared, bool reopen) {
     InitializeOptionsFromFlags(cache_, filter_policy_, options_);
   }
   InitializeOptionsGeneral(cache_, filter_policy_, sqfc_factory_, options_);
-
+  if (!strcasecmp(FLAGS_compression_manager.c_str(), "mixed")) {
+    // Currently limited to ZSTD compression. Table property compression_name
+    // needs to set to zstd for now even when there can be more than one
+    // algorithm in the table under your compressor.
+    options_.compression = kZSTD;
+    options_.bottommost_compression = kZSTD;
+    if (!ZSTD_Supported()) {
+      fprintf(stderr,
+              "ZSTD compression not supported thus mixed compression cannot be "
+              "used\n");
+      exit(1);
+    }
+    auto mgr = std::make_shared<RoundRobinManager>(
+        GetDefaultBuiltinCompressionManager());
+    options_.compression_manager = mgr;
+  }
```

Comment:
We should also abort if FLAGS_compression_manager is unrecognized. Empty and maybe "nullptr" can be treated as compression_manager = nullptr.